Makefile

```diff
@@ -48,21 +48,38 @@ fix: $(FIXERS)
 # END: lint-install ../malcontent
 
 SAMPLES_REPO ?= chainguard-dev/malcontent-samples
-SAMPLES_COMMIT ?= 75ee4b4f3e5c29484811e77ffa9d5d02eb75218f
+SAMPLES_COMMIT ?= f56db86f52ef9b647943d9e3b6652a85ccb68266
 OUT_DIR=out/samples-$(SAMPLES_COMMIT).tmp
 out/samples-$(SAMPLES_COMMIT):
 	mkdir -p out
 	git clone https://github.com/$(SAMPLES_REPO).git $(OUT_DIR)
 	git -C $(OUT_DIR) checkout $(SAMPLES_COMMIT)
-	find $(OUT_DIR) -name "*.xz" -execdir tar xJvf "{}" \;
+	@for file in $$(find $(OUT_DIR) -name "*.xz" -print0 | xargs -0 echo); do \
+    dir=$$(dirname "$$file"); \
+    base=$$(basename "$$file" .xz); \
+    fullpath="$$dir/$$base"; \
+    temp_path="$$fullpath".temp; \
+    xz -dkc "$$file" > "$$temp_path"; \
```

Comment:
just curious - why not `xz -d`? do you need the original files afterward? if so it'd save a fair bit of moving things around.

Reply:
True. In earlier iterations I was thinking the original needed to be left around if `tar` would work on it, but we don't need it now.

Removed in [`34e20b2` (#508)](https://github.com/chainguard-dev/malcontent/pull/508/commits/34e20b226ca3d8faa4f18e7dbf9f77b3bd959bc5).

I ran a full refresh to make sure nothing changed:
```
$ rm -rf out/; make prepare-samples; make refresh-sample-testdata
mkdir -p out
git clone https://github.com/chainguard-dev/malcontent-samples.git out/samples-f56db86f52ef9b647943d9e3b6652a85ccb68266.tmp
Cloning into 'out/samples-f56db86f52ef9b647943d9e3b6652a85ccb68266.tmp'...
remote: Enumerating objects: 396, done.
remote: Counting objects: 100% (68/68), done.
remote: Compressing objects: 100% (59/59), done.
remote: Total 396 (delta 15), reused 50 (delta 9), pack-reused 328 (from 1)
Receiving objects: 100% (396/396), 653.00 MiB | 105.35 MiB/s, done.
Resolving deltas: 100% (44/44), done.
git -C out/samples-f56db86f52ef9b647943d9e3b6652a85ccb68266.tmp checkout f56db86f52ef9b647943d9e3b6652a85ccb68266
Note: switching to 'f56db86f52ef9b647943d9e3b6652a85ccb68266'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at f56db86 Rename vitess directory (#11)
x bincapz
x wolfictl
x kuma-cp
x ./clickhouse
x ./vtadmin
x ./vtclient
x minio.x86_64
x mongosh
x caddy
x opa
x ./pull-scripts
x neuvector-agent.aarch64
x chezmoi
x pulumi
x trivy
mv out/samples-f56db86f52ef9b647943d9e3b6652a85ccb68266.tmp out/samples-f56db86f52ef9b647943d9e3b6652a85ccb68266
cp -a test_data/. out/samples-f56db86f52ef9b647943d9e3b6652a85ccb68266
mkdir -p out
go build -o out/mal ./cmd/mal
./test_data/refresh-testdata.sh ./out/mal out/samples-f56db86f52ef9b647943d9e3b6652a85ccb68266
regenerating test data, max_procs=8 ...
processing queue with length:      153 /var/folders/n6/xxn5d2zd3l1gghpx_7qppzs00000gn/T/tmp.UmSM9vpENY
test data regeneration complete!!

$ git diff
diff --git a/Makefile b/Makefile
index e1b12e6..02b3d7e 100644
--- a/Makefile
+++ b/Makefile
@@ -59,7 +59,7 @@ out/samples-$(SAMPLES_COMMIT):
     base=$$(basename "$$file" .xz); \
     fullpath="$$dir/$$base"; \
     temp_path="$$fullpath".temp; \
-    xz -dkc "$$file" > "$$temp_path"; \
+    xz -dc "$$file" > "$$temp_path"; \
     if file "$$temp_path" | grep -q "POSIX tar archive"; then \
       if [ "$(shell uname)" = "Darwin" ]; then \
         tar xJvf "$$temp_path" -C $$(dirname "$$temp_path"); \
```